config/default/kustomization.yaml

```diff
@@ -10,7 +10,7 @@ namePrefix: cartographer-conventions-
 
 # Labels to add to all resources and selectors.
 commonLabels:
-  component: conventions.carto.run
+  app.kubernetes.io/component: conventions.carto.run
```

Comment:
should we include `carto.run` in it? I think it makes sense to make it not as ambiguous as it would without it, but looking at some pods in the system

```console
$ k get pods -A -o yaml | grep component
      app.kubernetes.io/component: controller
      app.kubernetes.io/component: cainjector
      app.kubernetes.io/component: webhook
      component: conventions.apps.tanzu.vmware.com
      component: etcd
      component: kube-apiserver
      component: kube-controller-manager
      component: kube-scheduler
      app.kubernetes.io/component: controller
      app.kubernetes.io/component: webhook
                app.kubernetes.io/component: webhook
```

it doesn't look like a very common practice (although this is very little data 😅 )

Reply:
I don't have strong feelings about a prefix on the value. The important thing is to be consistent between all of the components that end up in package-for-cartographer